documentation/conf.py

```diff
@@ -27,6 +27,17 @@
 from recommonmark.transform import AutoStructify
 from recommonmark.parser import CommonMarkParser
 
+# -- Read the Docs --------------------------------------------------------
+
+# Define the canonical URL if you are using a custom domain on Read the Docs
+html_baseurl = os.environ.get("READTHEDOCS_CANONICAL_URL", "")
+
+# Tell Jinja2 templates the build is running on Read the Docs
+if os.environ.get("READTHEDOCS", "") == "True":
```

Comment:
Is there a particular reason to check for the value True rather than just check that a value is set? Usually we don't need such precision as being exactly the value "True" for something that only needs to be truthy 

suggestion:
```python
if os.environ.get("READTHEDOCS"): 
    # stuff
```

Reply:
This is taken verbatim from RTD's suggested config updates.

* https://about.readthedocs.com/blog/2024/07/addons-by-default/

Reply:
(But I agree the suggestion is cleaner)